examples/gatsby/BUILD.bazel

```diff
@@ -0,0 +1,25 @@
+load("@npm//gatsby-cli:index.bzl", "gatsby")
+
+gatsby(
```

Comment:
this has no outs[] or output_dir which means it doesn't do anything unless you `bazel run :build` - is that what you intend? how would you access the built artifacts? I would expect this target to be a dependency of a devserver or something

Reply:
Once I got the build working I would progress to that yes, at the minute can't get the build working.